analyzers/src/SonarAnalyzer.CSharp/Rules/MethodsShouldNotHaveTooManyLines.cs

```diff
@@ -86,5 +110,11 @@ protected override string GetMethodKindAndName(SyntaxToken identifierToken)
 
             return "method";
         }
+
+        private static long CountLines(IEnumerable<SyntaxNode> nodes) =>
```

Comment:
Would it be fairer to maybe distinguish between local functions and expressions in top-level statements?
Not for now of course (baby steps). But would it be worth opening an issue?

Reply:
For the moment I want to be consistent with the behavior for normal methods. I will open a discussion to see if we want to update the rule in this regard.